build-test.cmd

```diff
@@ -104,6 +104,13 @@ call %__ProjectDir%\init-tools.cmd
 
 REM =========================================================================================
 REM ===
+REM === Deploy debugger tests
```

Comment:
Also that way you can provide a knob to turn off these runs for non-ret builds of coreclr.